SPECS/nss_nis/nss_nis.spec

```diff
@@ -57,12 +57,38 @@ make check
 %{_unitdir}/systemd-logind.service.d/*
 %{_unitdir}/systemd-userdbd.service.d/*
 
-
 %license COPYING
```

Comment:
Small request: could you move `%license` to be the first entry under `%files`? Seems like our linter doesn't pick this up and that's the way we tend to keep it. Makes it easier to find and audit licensing info.

Reply:
Done.